### PR TITLE
Feature/0 install

### PR DESCRIPTION
**Pull request by napalmpapalam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+  </a>
+</div>
+
+# Rarimo JavaScript SDK
 TypeScript-based source development script for Rarimo Protocol
+
+## Changelog
+Change log can be found by the following link: [CHANGELOG.md]
+
+[CHANGELOG.md]: https://github.com/rarimo/js-sdk/blob/main/CHANGELOG.md
```
Comment (ihordiachenko) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+
+
+## API documentation
+
+The full API documentation is hosted at [rarimo.github.io/js-sdk]. This is a bit
```
Comment (ihordiachenko) [inline README.md:20]:
```suggestion
The full API documentation is hosted at [rarimo.github.io/js-sdk](https://rarimo.github.io/js-sdk). This is a bit
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,85 @@
...
+tricky to navigate and requires basic TypeScript understanding. It is helpful if
```
Comment (ihordiachenko) [inline README.md:21]: `This is a bit  tricky to navigate and requires basic TypeScript understanding.` - don't undermine your own stuff heh. Just say it's TypeDoc

File: README.md [hunk 4]
```diff
@@ -1,2 +1,85 @@
...
+you want to look up details for advanced use cases. This documentation is
+auto-generated based on the current main branch and can occasionally diverge
+from the latest release.
+
+[rarimo.github.io/js-sdk]: https://rarimo.github.io/js-sdk
```
Comment (ihordiachenko) [inline README.md:26]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,85 @@
...
+  <a href="https://github.com/rarimo/js-sdk/blob/main/LICENSE">
+    <img alt="License: MIT" src="https://img.shields.io/github/license/rarimo/js-sdk.svg" />
+  </a>
+</div>
+
+# Rarimo JavaScript SDK
 TypeScript-based source development script for Rarimo Protocol
+
+## Changelog
+Change log can be found by the following link: [CHANGELOG.md]
```
Comment (ihordiachenko) [inline README.md:13]:
```suggestion
Change log can be found by the following link: [CHANGELOG.md](https://github.com/rarimo/js-sdk/blob/main/CHANGELOG.md)
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,85 @@
...
+|----------------------------------------| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------|
+| [@rarimo/provider](packages/provider)  | A package which wraps managing multiple (EVM and non EVM type) chains wallet extension providers to the one common interface.| [![npm version](https://img.shields.io/npm/v/@rarimo/provider.svg)](https://www.npmjs.com/package/@rarimo/provider)       |
+| [@rarimo/nft-checkout](packages/nft-checkout) | A package with cross-chain NFT purchases functionality, based on Rarimo Protocol.| [![npm version](https://img.shields.io/npm/v/@rarimo/nft-checkout.svg)](https://www.npmjs.com/package/@rarimo/nft-checkout) |
+
+## Webpack Configs
+
+With WebPack 5, you have to be explicit about the usage of Node.js types and
```
[the remark this answers is not in the document]
Comment (napalmpapalam) [reply README.md:39]: it's config for webpack 5, for webpack 4 we don't need to do anything. just missed to replace `: ` to `.`

**Pull request merged**